poetry.lock

```diff
@@ -1,15 +1,15 @@
-# This file is automatically @generated by Poetry and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.4.2 and should not be changed by hand.
```

Comment:
What's happening here? We are all on version `1.3.2`, I'm worried this might break something in the poetry lock

Reply:
Fixed, but we need to probably update to latest poetry,